### PR TITLE
Test commit

### PR DESCRIPTION
**Pull request by DayMax86**
Test

### CODE REVIEW

**Comment by DayMax86** [comment]
Don't be alarmed, all the changes are just VS Code doing auto formatting - I only added a line into a comment block.

File: public/index.html [hunk 1]
```diff
@@ -39,5 +39,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```
Comment (kt902) [inline public/index.html:44]: Please make sure there's always a newline at the end of the file. 😱

File: public/index.html [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
```
Comment (kt902) [inline public/index.html:5]: The formatting here puts the `html` and `head` and `body` tags at the same level of indentation, which is not desirable. Any idea what caused that ?

**Review by kt902 — COMMENTED** [review]
Great to see the first PR roll in. Excited for what the future holds. Left some comments

**Comment by kt902** [comment]
Closing out test PR

**Pull request closed without merging**